main.go

```diff
@@ -16,15 +16,18 @@ import (
 	"syscall"
 )
 
-var flagSet = flag.NewFlagSet("dockexec", flag.ContinueOnError)
+var (
+	flagSet  = flag.NewFlagSet("dockexec", flag.ContinueOnError)
+	fCompose = flagSet.Bool("compose", false, "use docker-compose instead of docker to run the resulting binary")
```

Comment:
Ah, I intentionally used "resulting" here because use of `dockexec` is not specific to `go test`. Indeed I'm using it in the context of `go run`. Thoughts?

Reply:
Perhaps the alternative use case of `go run` can be addressed in a further PR, because most (all?) of the `dockexec` docs refer to `go test`. I'll make this change for now in favour of making the docs more generic at a later date.

Reply:
Agreed, makes sense.